
Do not call depmod (especially not without the currently building version
attached) as well as not injecting some device nodes ...

  - Rene Rebe <rene@exactcode.de>

--- NVIDIA-Linux-x86-1.0-6629-pkg1/usr/src/nv/Makefile.nvidia~	2005-01-06 00:29:31.718683968 +0100
+++ NVIDIA-Linux-x86-1.0-6629-pkg1/usr/src/nv/Makefile.nvidia	2005-01-06 00:30:57.207687664 +0100
@@ -112,9 +112,6 @@
 package-install: module suser-sanity-check rmmod-sanity-check 
 	mkdir -p $(MODULE_ROOT)/video && \
 	install -m 0664 -o root -g root nvidia.o $(MODULE_ROOT)/video/nvidia.o$(O) && \
-	PATH="$(PATH):/bin:/sbin" depmod -a && \
-		PATH="$(PATH):/bin:/sbin" modprobe nvidia && \
-	sh makedevices.sh && \
 	echo "nvidia.o installed successfully."; \
 
 $(VERSION_HDR): 
